README.rst

```diff
@@ -196,9 +196,10 @@ If not you can skip those playbook and proceed onto `deploy-openstack-config` wh
 Be sure to replace `ansible_user` with the user you are using to connect to the Ansible control host.
 .. code-block:: console
 
-   ansible-playbook -i $(terraform output -raw ansible_control_access_ip_v4), ansible/fix-homedir-ownership.yml -e ansible_user=cloud-user
-   ansible-playbook -i $(terraform output -raw ansible_control_access_ip_v4), ansible/grow-control-host.yml -e ansible_user=cloud-user
-   ansible-playbook -i $(terraform output -raw ansible_control_access_ip_v4), ansible/deploy-openstack-config.yml -e ansible_user=cloud-user
+   ansible-playbook -i ansible/inventory.yml ansible/fix-homedir-ownership.yml -e ansible_user=cloud-user
```

Comment:
With this list growing the way it is, is it worth creating one playbook that imports the others?